renderer/reducers/neutrino.js

```diff
@@ -108,17 +108,18 @@ export const startNeutrino = lndConfig => async dispatch => {
     // Initialise the Neutrino Web Worker..
     const neutrino = await neutrinoService
     await neutrino.init(lndConfig)
+    dispatch(initNeutrino())
 
     // Start the service and wait for one of the gRPC interfaces to become active.
     await new Promise(async resolve => {
-      neutrino.once(
+      neutrino.on(
```

Comment:
Do we ensure this is cleared to prevent double subs?

Reply:
I've added calls to remove these listeners in the `NEUTRINO_EXIT` event handler.